src/workloads/query/query_stats/BaseFixed.yml

```diff
@@ -0,0 +1,41 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This workload tests for performance regressions without concurrent $queryStats aggregates,
+  using fixed-size payloads in the find queries.
+
+KeyWords:
+- queryStats
+- redaction
+
+Clients:
+  Default:
+    QueryOptions:
+      connectTimeoutMS: 3_600_000  # 1 hour
+      maxPoolSize: 4000
+      socketTimeoutMS: 3_600_000  # 1 hour
+
+GlobalDefaults:
+  # Workload "constants."
+  NumGroups: &NumGroups 1  # Excluding the setup group.
+
+Actors:
+  ^FlattenOnce:
+  - LoadConfig:
+      Path: ../../../phases/query/QueryStatsWorkloads.tmpl
+      Key: SetupGroup
+      Parameters:
+        NumGroups: *NumGroups
+  - LoadConfig:
+      Path: ../../../phases/query/QueryStatsWorkloads.tmpl
+      Key: AllBaseFixedGroup
+      Parameters:
+        NumGroups: *NumGroups
+        Group: 1
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - standalone-query-stats
```

Comment:
Can we run this on replica sets as well? And maybe also sharded clusters? I think this is a good template list: https://github.com/mongodb/genny/blob/788eaf26e3b8b08d76c71d54fb0013befee9b032/src/workloads/query/TenMDocCollection_IntId.yml#L188 - would that work? Any concerns?